### DISCUSSION TITLE
I can't ping/access all clients via WGeasy

### DISCUSSION THREAD

**Discussion by Wolf6660** [wg-easy/wg-easy#1487] [Q&A]
Hello,

I am installing WGEasy with the default settings

docker run --detach \
  --name wg-easy \
  --env LANG=de \
  --env WG_HOST=mydnsdnsadress \
  --env PASSWORD_HASH=mypw \
  --env PORT=51821 \
  --env WG_PORT=51820 \
  --volume ~/.wg-easy:/etc/wireguard \
  --publish 51820:51820/udp \
  --publish 51821:51821/tcp \
  --cap-add NET_ADMIN \
  --cap-add SYS_MODULE \
  --sysctl 'net.ipv4.conf.all.src_valid_mark=1' \
  --sysctl 'net.ipv4.ip_forward=1' \
  --restart unless-stopped \
  ghcr.io/wg-easy/wg-easy

Now when I connect the VPN, I can access the router (192.168.2.1) and the VPN server (192.168.2.2). I cannot connect to my NAS (192.168.2.10) and the IP cam 192.168.2.22.

Does anyone have any ideas?

**Comment by DecoAri** [comment]
Do u use docker desktop or orbstack?

These day I was looking for a solution to let wg-easy work in orbstack.

For now. wg-easy can only work in Parallels desktop and VMfusion.

In orbstack. Traffic can through the lan. all clents can send and receive traffic but there is no internet. But. Shadowsocks can work in orbstack, Network type: host. wg-easy network type: bridge

**Comment by kaaax0815** [comment]
This seems like a firewall issue. As there isn't any reason why it wouldn't work
